### ISSUE TITLE
Crash When Selecting 8th Tradition

### ISSUE THREAD

**Issue by TheNoremac95** [OriginalTrinity/More-Ascension-Perk-Slots#10]
I have seven traditions filled out, and I was selecting the eighth, but when I clicked the "select" button the game crashed. It looks like the most common reason for this issue is in load order, particularly with UI mods, so I fiddled with that a bit and but it as low as I could without breaking the game entirely. However, it's still crashing. Hope you can help!

[error.log](https://github.com/OriginalTrinity/More-Ascension-Perk-Slots/files/8257936/error.log)

Load Order:

Beautiful Universes v2.0
Pony Mod
Gigastructural Engineering & More (3.3)
UI Overhaul 1080p Plus
Real Space 3.8
No Clustered Starts
Dynamic Political Events
Planetary Diversity
More Trait Points
**More Ascension Perk Slots**
Stellaris Evolved - Alpha        <--- Needs to be at the bottom or game crashes on load-up.

**Comment by OriginalTrinity** [comment]
You're right usually this is caused by UI Mods like UI Overhaul. But in this case, it's Stellaris Evolved which is the troublemaker.
It is in direct conflict on multiple occasions. Most of them don't have any negative impact except one. Both mods alter the design of the tradition trees. This means that if my mod gets overridden, the custom tree templates I used and defined won't exist and will cause the game to crash. -> `[18:49:28][gui.cpp:1074]: Tried to get gui_type [tree_11_12_21] which does not exist`

A compatibility patch is needed for both mods to work properly. I got some spare time so I'll do it myself.
